api-gateway/src/main/resources/application.yml

```diff
@@ -58,47 +82,37 @@ spring:
     encoding:
       force: true
 
+# Set the context-path. WebFlux calls it base-path
+# https://stackoverflow.com/questions/49196368/context-path-with-webflux
+# Kubernetes rewrites the url to use `/`
+spring.webflux.base-path: /
```

Comment:
This is the default value, but I want to make this setting explicit since Spring WebFlux uses a different configuration setting than Spring MVC.